src/languages/en.ts

```diff
@@ -1382,7 +1382,7 @@ export default {
             conditionsDetails: 'Find details and conditions for all fees and services by visiting',
             conditionsPhone: 'or calling +1 833-400-0904.',
             instant: '(instant)',
-            electronicFundsInstantFeeMin: '(min $0.25)',
+            electronicFundsInstantFeeMin: ({amount}: TermsParams) => `(min ${amount})`,
```

Comment:
hmmmm should we call the amount formatter directly from here so you can just pass a number to this and not have to format it on every caller?
Can we even do that?

Applies to the rest too

Reply:
I can do it with `CurrencyUtils.convertToDisplayString` but not with `numberFormat` so I can change a few of them

Reply:
Hmm actually, it does not work unless I pass the locale to `CurrencyUtils.convertToDisplayString` since it's not using useLocalize and so the locale is off.

Reply:
@iwiznia Do you want to complete the checklist or do you want me to add a C+ to test this as well?